component-library/src/app/components/email-verification/email-verification.component.html

```diff
@@ -0,0 +1,103 @@
+<div
+  class="email-verification"
+  style="margin: 32px 0 0 0"
+>
+  <ng-container *ngIf="status === 'start'; else waiting">
+    <div>
+      <p>
+        {{ 'Please verify your mail before submitting the form.' | translate }}
+      </p>
+      <div style="display: grid; grid-template-columns: 50% 50%">
```

Comment:
@MikeJalow Are we supposed to avoid using inline css?

Reply:
Yup! Good catch. If at all possible it'd be good if we could do this through either Core or one of the stylesheets we have.

Reply:
That being said it's not really a component shipped with our library right so I figure I'll leave it to Alexandre's discretion

Reply:
Oh yeah that's on me, I meant to clean it up after I was done, but I forgot to do it. Good catch.